files/default/sql/ddl/0.6.1.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `hopsworks`.`project` ADD COLUMN `conda_env` TINYINT(1) DEFAULT 0;
```

Comment:
Shouldn't this be `default 1`? On the current installations each project has a conda_env already